src/dashboard/DashboardThunks.ts

```diff
@@ -303,70 +303,71 @@ export const loadDashboardFromNeo4jThunk = (driver, database, uuid, callback) =>
 
   try {
     const query = 'MATCH (n:_Neodash_Dashboard) WHERE n.uuid = $uuid RETURN n.content as dashboard';
```

Comment:
We should manually check that all these operations (dashboard load via share link, dashboard load via sidebar, dashboard load in readonly mode) are all still functional. There's quite some breaking points that we should double check before pushing out the next release.